setup.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS responses (
+  id    SERIAL PRIMARY KEY,
+  added_date    timestamp WITH time zone,
+  survey_response    jsonb
+);
```

Comment:
Depends what the retrieval patterns are - Postgres will automatically create an index on the primary key, so other than retrieval by "id", what queries do we use?